scripts/check-and-run-apollo-cli.sh

```diff
@@ -17,6 +17,10 @@ if [[ -z "$CONFIGURATION" ]]; then
   exit 1
 fi
 
+if [[ -x /opt/local/bin/port ]]; then
```

Comment:
Good catch! Mind throwing in a comment here so the next person spelunking knows what this is for?